<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,0">
      <item>
       <widget class="QSlider" name="horizontalSliderTime">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditTime"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditTimeNow">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelInQueue">
            <property name="text">
             <string>入口排队</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelInQueueCnt">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>0/0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditInQueue"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelEmpltSlotCnt">
            <property name="text">
             <string>空车位数</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditEmptySlotsCnt">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="GraphicsViewCarPark" name="graphicsViewCarPark"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelOutQueue">
            <property name="text">
             <string>出口排队</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelOutQueueCnt">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditOutQueue"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>停车记录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="topMargin">
           <number>23</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBoxRandom">
            <property name="text">
             <string>随机车牌</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxAutoIndex">
            <property name="text">
             <string>自动车位号</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinBoxTotalSize">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>128</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelTotalSize">
            <property name="text">
             <string>总车位</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditLicense"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelLicense">
            <property name="text">
             <string>车牌</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBoxQueueSize">
            <property name="maximum">
             <number>128</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelQueueSize">
            <property name="text">
             <string>队列长度</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelIndex">
            <property name="text">
             <string>车位号</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEditIndex"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAdd">
          <property name="text">
           <string>加入车辆</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionExport"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsViewCarPark</class>
   <extends>QGraphicsView</extends>
   <header location="global">carpark_graphics_view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
